packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.js

```diff
@@ -55,7 +55,7 @@ Meteor.methods({
 		if (data.language !== 'en') {
 			const localeFn = Meteor.call('loadLocale', data.language);
 			if (localeFn) {
-				Function(localeFn)();
+				Function(moment.localeFn)();
```

Comment:
@ggazzo any ideas on this?  I know this file was converted from coffeescript to js.  So I can't see who wrote the line before or what it used to look like here :grin: